src/main/java/com/devshop/devshop/service/DevshopService.java

```diff
@@ -0,0 +1,21 @@
+package com.devshop.devshop.service;
+
+import com.devshop.devshop.model.Category;
+import com.devshop.devshop.repository.CategoryRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class DevshopService {
+    private final CategoryRepository categoryRepository;
+
+    public DevshopService(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+
+    public List<Category> getCategoriesList() {
```

Comment:
Dobra praktyka jest nazywanie metod w serwisie tak jak w repo, wiec skoro masz w repository nazwe metody `findAll()` to fajnie by bylo jakby w serwisie metoda tez byla `findAll()` :)